app/assets/stylesheets/sidebar-data.scss

```diff
@@ -16,3 +16,11 @@
     border-radius: 3px;
   }
 }
+
+.daily-stats {
```

Comment:
Good job and changing up the styling to match the other sections in the dashboard above this section!